technical-reports/color/index.html

```diff
@@ -80,9 +80,17 @@ <h1>Introduction</h1>
         <em>This section is non normative</em>
       </p>
 
-      <p>Work in progress.</p>
+      <section
+        data-include="./overview.md"
+        data-include-format="markdown"
+      ></section>
     </section>
 
+    <section
+      data-include="./token-naming.md"
+      data-include-format="markdown"
```

Comment:
```suggestion
      data-include-format="markdown"
      class="informative"
```

^ Denotes this section is non-normative.